src/capwap_loc_provider_sup.erl

```diff
@@ -33,6 +33,16 @@
 start_link(Config) ->
     supervisor:start_link({local, ?MODULE}, ?MODULE, [Config]).
 
+location_children() ->
+    case application:get_env(location_provider) of
```

Comment:
```suggestion
    case application:get_env(capwap, location_provider) of
```

Reply:
Hum. I don't agree here. The less dependencies we add here, the better, in case the code needs to be moved somewhere else at some point. It's under the capwap application, so it will search in the capwap env vars.

Reply:
Then why you use same approach in other places?

Reply:
In any case added at https://github.com/travelping/capwap/pull/63 because expected that `location_provider` is a part of config of `capwap` - please see the CT example https://github.com/travelping/capwap/blob/master/test/capwap_http_api_SUITE.erl#L283